### ISSUE TITLE
Improper spelldamage Frostfirebolt, Firebolt

### ISSUE THREAD

**Issue by Vidor94** [dalaranwow/dalaran-wow#5525]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

I always found it weired how little dmg the frostfirespecc does in raids so i wanted to test it out if the Scaling works properly.
So I calcualted the Theoretical Damage for Frostfirebolt and Firebolt with the Spellpowercoeficient of http://wowwiki.wikia.com/wiki/Spell_power_coefficient.
I only took a loot at the initial Damage not the Dot from the Spells. 
The Formula I used for the spell dmg is: Spellpower*Spellpowercoefficient+Base damage
I took the Spellpowercoeficient from the Wiki (2 lines up) and the Base Damage from the ingame tooltip, I took the average base damage = (smallest nr+highest nr)/2

Next i put in all the dmg Modifiers from talent and in case of Frostfirebolt from the glyph. 

**Expected behaviour**: Tell us what should happen instead
In the Calcualtion (Excel in attachment) the Dmg for the noncrits is for both about 2.5% too low and and the crits for Frostfirebolt 7.5% too low and 0.91% too high for Firebolt. From the noncrit the the critical dmg the error should increase since it gets multiplied, but in both cases this is not the case. The 2.5% don't translate into 7.5% of the decrease of the crit and the error doesn't just disappear because it crit for the firebolt. 
It is also very interesting that in both cases the highest noncrit does not reach the average theoretical damage, so there is for sure an error and it's not due to the rather average sample size.
Now where excatly is the Problem? i don't know but it is there, i plan to make some more testing, but well it's just a plan. This error makes Frostfirebolt ofc very bad, since the critical dmg from frostfirebolt in the reason why you are picking in and when it's too low it's just now worth giving up that much.

If there is any error in my calcualtion or i forgot something pls tell me. 

**Steps to reproduce the problem**:

1.  go to a puppet
2.  make sure your spellpower is stable. (critproccs or haste proccs don't matter for this)
3.  fire away

I kept most of the gear on, since i wanted to have a good number of crits just took of the spellpowerproc trinket and ring from icc. The sample size is not huge but both spell have over 100 instances so good enough to see there is a problem. 

**Include proofs for this behaviour**

![image](https://user-images.githubusercontent.com/38830771/41775052-8a187704-7622-11e8-9503-384323dadf7b.png)

[Damage Calculation Mage Spells.xlsx](https://github.com/dalaranwow/dalaran-wow/files/2127482/Damage.Calculation.Mage.Spells.xlsx)
http://wowwiki.wikia.com/wiki/Spell_power_coefficient

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wowgaming.altervista.org/aowow/?spell=47610
https://wowgaming.altervista.org/aowow/?spell=42833

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![image](https://user-images.githubusercontent.com/38830771/41775052-8a187704-7622-11e8-9503-384323dadf7b.png)
